tests/yast2_gui/yast2_network_settings.pm

```diff
@@ -17,66 +17,76 @@
 use base "y2x11test";
 use strict;
 use testapi;
+use version_utils 'is_leap';
 
 sub run {
     my $self = shift;
-    $self->launch_yast2_module_x11('lan', match_args => [qw(yast2-lan-ui yast2_still_susefirewall2)], match_timeout => 60);
-    if (match_has_tag('yast2_still_susefirewall2')) {
+
+    # keyboard shorcuts
+    $cmd{global}          = 'alt-g';
+    $cmd{dhcp_client_id}  = 'alt-i';
+    $cmd{overview}        = 'alt-v';
+    $cmd{add_device}      = 'alt-a';
+    $cmd{dynamic_address} = 'alt-y';
+    $cmd{hostname_dns}    = 'alt-s';
+    $cmd{routing}         = 'alt-u';
+
+    $self->launch_yast2_module_x11('lan', target_match => [qw(yast2-lan-ui yast2_still_susefirewall2)], match_timeout => 60);
+    if (match_has_tag 'yast2_still_susefirewall2') {
         record_soft_failure "bsc#1059569";
         send_key 'alt-i';
         wait_still_screen;
     }
-
-    #	Global Options
```

Comment:
Could you please also put new or recover old comment, I believe they are helpful even if you have introduced better named variables.

Reply:
yes, it is true that for separating the tab makes sense, I wrote new ones, but definitively there are many useless in these tests in general.